dockers/docker-database/Dockerfile.j2

```diff
@@ -60,4 +62,6 @@ COPY ["critical_processes", "/etc/supervisor"]
 COPY ["files/update_chassisdb_config", "/usr/local/bin/"]
 COPY ["flush_unused_database", "/usr/local/bin/"]
 
+LABEL com.azure.sonic.manifest="$manifest"
+
 ENTRYPOINT ["/usr/local/bin/docker-database-init.sh"]
```

Comment:
What about adding its version as "label" ?
[SB] What will be the usage of this label?

Curious! Can we move this to base docker file (docker-config-engine) ?
[SB] We can, althoug if we want to avoid duplicating it I would rather do it as --label argument to docker build. I'll do the change.